pyproject.toml

```diff
@@ -1,11 +1,53 @@
 [build-system]
 requires = [
-  "setuptools",
-  "setuptools_scm[toml]",
+  "setuptools>=62.1",
+  "setuptools_scm[toml]>=6.2",
   "wheel",
   "oldest-supported-numpy",
```

Comment:
I don't think we need this here. @nabobalis there's no C extension in this repo right?

```suggestion
```

Reply:
Yeah, no need. I did it for fun